### ISSUE TITLE
Don't get how the "group" keyword works

### ISSUE THREAD

**Issue by aruhier** [Mic92/python-mpd2#53]
Hello,
I'm trying to list my albums grouped by artists, but I don't really get how to do that : your documentation indicates that list() can take an argument "group", so I try to do that:

```
    mpdclient.connect(server, port)
    mpdclient.list("album", "group", "albumartist")
mpd.CommandError: [2@0] {list} not able to parse args
```

Did I misunderstood your doc or do I have to do this in 2 operations (listing artists, and then listing the albums for each artist of the list) ?
Thank you

**Comment by mxjeff** [comment]
This is related to #50, I did not noticed grouping was available for list command as well /o\

The solution might be different though.

**Comment by ibeex** [comment]
Any info on this, it still don't work.

**Comment by rnixx** [comment]
Fixed via #106
